readthedocs/settings/base.py

```diff
@@ -113,9 +113,12 @@ def SESSION_COOKIE_SAMESITE(self):
     CSRF_COOKIE_HTTPONLY = True
     CSRF_COOKIE_AGE = 30 * 24 * 60 * 60  # 30 days
 
+    # Remove after deploying
```

Comment:
```suggestion
    # TODO: Remove after deploying
```

Reply:
We should also make an issue to track this to make sure we do it, or a PR that does it, that we merge the week after.

Reply:
Good point about opening a PR directly 💯 ; https://github.com/readthedocs/readthedocs.org/pull/10608